disnake/threads.py

```diff
@@ -975,3 +1095,153 @@ def _from_data(self, data: ThreadMemberPayload):
     def thread(self) -> Thread:
         """:class:`Thread`: The thread this member belongs to."""
         return self.parent
+
+
+class ForumTag(Hashable):
+    """
+    Represents a tag for threads in forum channels.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two tags are equal.
+
+        .. describe:: x != y
+
+            Checks if two tags are not equal.
+
+        .. describe:: hash(x)
+
+            Returns the tag's hash.
+
+        .. describe:: str(x)
+
+            Returns the tag's name.
+
+    .. versionadded:: 2.6
+
+
+    Examples
+    --------
+
+    Creating a new tag:
+
+    .. code-block:: python3
+
+        tags = list(forum.available_tags)
```

Comment:
this already occurs so we (or other users) don't need to use `list()` beforehand.

Reply:
https://github.com/DisnakeDev/disnake/pull/726/commits/1667b98f7e197ffb689d51996a716a0a399fcb85